Libplanet/KeyStore/Kdfs/Scrypt.cs

```diff
@@ -0,0 +1,250 @@
+using System;
+using System.Collections.Immutable;
+using System.Diagnostics.CodeAnalysis;
+using System.Diagnostics.Contracts;
+using System.Linq;
+using System.Text.Json;
+using Org.BouncyCastle.Crypto.Digests;
+
+namespace Libplanet.KeyStore.Kdfs
+{
+    /// <summary>
+    /// <a href="https://en.wikipedia.org/wiki/Scrypt">Scrypt</a>.
+    /// </summary>
+    /// <typeparam name="T">PRF (pseudorandom function) to use, e.g.,
+    /// <see cref="Sha256Digest"/>.</typeparam>
+    [SuppressMessage(
+        "Microsoft.StyleCop.CSharp.ReadabilityRules",
+        "SA1402",
+        Justification = "There are just generic & non-generic versions of the same name classes.")]
+    [Pure]
+    public sealed class Scrypt<T> : IKdf
```

Comment:
It seems this type parameter `T` is not used in this class. Do we need this?

Reply:
That's a breadcrumb when I copied and pasted from Pbkdf2, and Norgerman.Cryptography.Scrypt use SHA256 digest internally, so there are no reason to use <T> in this class. I'll remove it.